pymfe/mfe.py

```diff
@@ -1241,37 +1241,77 @@ def parse_by_group(
 
         return tuple(filtered_res)
 
+    @staticmethod
+    def _parse_description(docstring: str) -> t.Tuple[str, str]:
+        """Parse the docstring to get initial description and reference.
+
+        Parameters
+        ----------
+        docstring : str
+            An numpy docstring as ``str``.
+
+        Returns
+        -------
+        tuple of str
+            The initial docstring description in the first position and the
+            reference in the second.
+
+        """
+        initial_description = ""  # type: str
+        reference_description = ""  # type: str
+
+        # get initial description
+        split = docstring.split("\n\n")
+        if split:
+            initial_description = " ".join(split[0].split())
+
+        # get reference description
+        aux = docstring.split("References\n        ----------\n")
+        if len(aux) >= 2:
+            split = aux[1].split(f".. [")
+            if len(split) >= 2:
+                del split[0]
+                for spl in split:
+                    reference_description += "[" + " ".join(
+                        spl.split()) + "\n"
+
+        return (initial_description, reference_description)
+
     @classmethod
     def metafeature_description(
             cls,
             groups: t.Optional[t.Union[str, t.Iterable[str]]] = None,
-            sort: bool = False,
-            print_table: bool = True) -> t.Optional[t.List[t.List[str]]]:
+            sort_by_group: bool = False,
+            sort_by_mtf: bool = False,
+            print_table: bool = True
```

Comment:
From a scientific point of view, it is highly interesting to have the references as part of the table entries. But for a 'regular user' maybe it can make the table get cumbersome too easily? Maybe we could create a boolean argument to 'include_references: bool = True' (or, equivalently, 'suppress_references: bool = False') to enable the user choose whether he or she actually wants the references while using the library?

Reply:
I agree. I added a new parameter called include_references in this commit b9d81bc